test/connectivity/modules/NetworkController/FirewallOfflineController.cs

```diff
@@ -10,54 +9,37 @@ namespace NetworkController
 
     class FirewallOfflineController : IController
     {
-        static readonly ILogger Log = Logger.Factory.CreateLogger<FirewallOfflineController>();
-        readonly IFirewallCommands networkCommands;
+        static readonly ILogger Log = Logger.Factory.CreateLogger<LinuxFirewallOfflineController>();
+        readonly IController underlyingController;
 
         public FirewallOfflineController(string networkInterfaceName)
         {
-            Preconditions.CheckNonWhiteSpace(networkInterfaceName, nameof(networkInterfaceName));
```

Comment:
it is no harm to validate here, why remove?

Reply:
it's not harm, but since this class is internal it's not needed

---
In reply to: [359155265](https://github.com/Azure/iotedge/pull/2137#discussion_r359155265) [](ancestors = 359155265)